src/index.html

```diff
@@ -7,8 +7,46 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="nav">
+        <a href="index.html" class="logo"><img src="images/logo.png" alt="logo"></a>
+
+        <ul class="nav__links">
+          <div class="rectangle"></div>
+          <li class="nav__li">
```

Comment:
You need to save the style code and here you need to add an empty ribbon between the children of the elements of the same block, this also applies to all similar cases in the subsequent code